packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -559,7 +534,10 @@ export default class TradeStore extends BaseStore {
                     }),
                 });
                 return;
+            } else if (!is_on_mf_account) {
+                showUnavailableLocationError(showError, is_logged_in);
             }
+            showUnavailableLocationError(showError, is_logged_in);
```

Comment:
@matin-deriv `showUnavailableLocationError` is going to be called irrespective of the `is_on_mf_account` condition as it's also being called outside the `else` block here

Also if `is_on_mf_account` is `false`, `showUnavailableLocationError` will be called twice